docs/use_cases/met_tool_wrapper/METdbLoad/METdbLoad.py

```diff
@@ -0,0 +1,116 @@
+=========================
+
+met_tool_wrapper/METdbLoad/METdbLoad.conf
+
+"""
+##############################################################################
+# Scientific Objective
+# --------------------
+#
+# Load MET data into a METviewer database using the met_db_load.py script
+# found in dtcenter/METdatadb
+
+##############################################################################
+# Datasets
+# --------
+#
+# | **Input:** Various MET .stat and .tcst files
+#
+# | **Location:** All of the input data required for this use case can be found in the met_test sample data tarball. Click here to the METplus releases page and download sample data for the appropriate release: https://github.com/dtcenter/METplus/releases
+# | This tarball should be unpacked into the directory that you will set the value of INPUT_BASE. See `Running METplus`_ section for more information.
+# |
+
+##############################################################################
+# METplus Components
+# ------------------
+#
+# This use case utilizes the METplus METdbLoad wrapper to search for
+# files ending with .stat or .tcst, substitute values into an XML load
+# configuration file, and call met_db_load.py to load MET data into a
+# METviewer database.
+
+##############################################################################
+# METplus Workflow
+# ----------------
+#
+# METdbLoad is the only tool called in this example. It does not loop over
+# multiple run times:
+#
+
+##############################################################################
+# METplus Configuration
+# ---------------------
+#
+# METplus first loads all of the configuration files found in parm/metplus_config,
+# then it loads any configuration files passed to METplus via the command line
+# with the -c option, i.e. -c parm/use_cases/met_tool_wrapper/METdbLoad/METdbLoad.conf
+#
+# .. highlight:: bash
+# .. literalinclude:: ../../../../parm/use_cases/met_tool_wrapper/METdbLoad/METdbLoad.conf
+
+##############################################################################
+# XML Configuration
+# -----------------
+#
+# METplus substitutes values in the template XML configuration file based on
+# user settings in the METplus configuration file. While the XML template may
+# appear to reference environment variables, this is not actually the case.
+# These strings are used as a reference for the wrapper to substitute values.
+#
+# .. note::
+#     See the :ref:`METdbLoad XML Configuration<met_db_load-xml-conf>`
+#     section of the User's Guide for more information on the values
+#     substituted in the file below:
+#
+# .. highlight:: bash
+# .. literalinclude:: ../../../../parm/use_cases/met_tool_wrapper/METdbLoad/METdbLoadConfig.xml
+
+##############################################################################
+# Running METplus
+# ---------------
+#
+# This use case can be run two ways:
+#
+# 1) Passing in METdbLoad.conf then a user-specific system configuration file::
```

Comment:
```suggestion
# 1) Passing in METdbLoad.conf followed by a user-specific system configuration file::
```

Reply:
This text along with the text above about sample data are found in every use case documentation file. I agree with these changes, but we should apply this to all use case docs. There is an open GitHub issues to clean up these docs, but it was moved to future versions because we have already gone over our budget for documentation this cycle. We should add this info to the comments in that issue for when we are able to tackle it: https://github.com/dtcenter/METplus/issues/681